examples/src/main/kotlin/usageExamples/insert/InsertOne.kt

```diff
@@ -0,0 +1,45 @@
+package usageExamples.insert
+
+// :replace-start: {
+//    "terms": {
+//       "CONNECTION_URI_PLACEHOLDER": "\"<connection string uri>\"",
+//       "import io.github.cdimascio.dotenv.dotenv\n": ""
+//    }
+// }
+// :snippet-start: insert-usage-example
+import com.mongodb.MongoException
+import com.mongodb.kotlin.client.coroutine.MongoClient
+import io.github.cdimascio.dotenv.dotenv
+import kotlinx.coroutines.runBlocking
+import org.bson.codecs.pojo.annotations.BsonId
+import org.bson.types.ObjectId
+
+data class Movie(@BsonId val id: ObjectId, val title: String, val genres: List<String>)
```

Comment:
why not use the same data class as insertMany() example here?

Reply:
You could do. It probably was simplified in insertMany example due to the verboseness of Java.